pkg/trace/info/stats_test.go

```diff
@@ -0,0 +1,46 @@
+package info
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestTracesDroppedStatsToMap(t *testing.T) {
```

Comment:
Here it's fine to create 2 tests with 2 sub-tests each:
* `TestTracesDropped` with sub-tests `StatsToMap` and `StatsToString`
* `TestTracesMalformed` with sub-tests `StatsToMap` and `StatsToString`

Reply:
Yeah I guess that's a bit nicer here. Updated.